hw/acpi/pcihp.c

```diff
@@ -87,13 +87,13 @@ static void *acpi_set_bsel(PCIBus *bus, void *opaque)
 
 static void acpi_set_pci_info(AcpiPciHpState *s)
 {
-    static bool bsel_is_set;
     unsigned bsel_alloc = ACPI_PCIHP_BSEL_DEFAULT;
 
-    if (bsel_is_set) {
+    /* To avoid setting the same property again */
```

Comment:
@liujing2 where is this property used?

Reply:
This is used by acpi_pcihp_reset, for each pci root bus.